### PR TITLE
feat(payment): PAYPAL-553 3ds for PayPal

### PR DESCRIPTION
**Pull request by helga1507**
## What?
Add checking 3ds for PayPal Commerce

## Testing / Proof
https://www.loom.com/share/cd20ce2ddc7a4b1bbf68400135cd5fbd

@bigcommerce/checkout @bigcommerce/payments

### CODE REVIEW

**Review by BC-ACherednichenko — APPROVED** [review]
Nicely done

File: src/payment/strategies/paypal-commerce/paypal-commerce-credit-card-payment-strategy.ts [hunk 1]
```diff
@@ -48,7 +52,7 @@ export default class PaypalCommerceCreditCardPaymentStrategy implements PaymentS
 
         await this._store.dispatch(this._orderActionCreator.submitOrder(order, options));
 
-        const { orderId } = await this._paypalCommerceHostedForm.submit();
+        const { orderId } = await this._paypalCommerceHostedForm.submit(this._is3dsEnabled);
```
Comment (davidchin) [inline src/payment/strategies/paypal-commerce/paypal-commerce-credit-card-payment-strategy.ts:55]: You can get the value directly from the state so you don't need to store a temporary variable and having to unset it later. 🍹  i.e.:

```ts
const { paymentMethods: { getPaymentMethodOrThrow } } = await this._store.dispatch(this._orderActionCreator.submitOrder(order, options));
const { orderId } = await this._paypalCommerceHostedForm.submit(getPaymentMethodOrThrow().config.is3dsEnabled);
```

**Review by davidchin — APPROVED** [review]
LGTM 👍

File: src/payment/strategies/paypal-commerce/paypal-commerce-credit-card-payment-strategy.ts (hunk 1 above)
Comment (helga1507) [reply src/payment/strategies/paypal-commerce/paypal-commerce-credit-card-payment-strategy.ts:55]: Thank you!

**Pull request merged**